src/main/java/de/picturesafe/search/elasticsearch/impl/SingleIndexElasticsearchServiceImpl.java

```diff
@@ -30,7 +31,7 @@
 
     @Autowired
     public SingleIndexElasticsearchServiceImpl(ElasticsearchService elasticsearchService,
-                                               SingleIndexPresetConfigurationProvider indexPresetConfigurationProvider) {
+                                               IndexPresetConfigurationProvider indexPresetConfigurationProvider) {
         this.elasticsearchService = elasticsearchService;
         // Compatibility to ElasticsearchServiceImpl
         this.indexPresetConfiguration = indexPresetConfigurationProvider.getIndexPresetConfiguration("default");
```

Comment:
Now the name should be configurable, because the code cannot rely on the fact that it is an instance of SingleIndexPresetConfigurationProvider anymore. Something like:

@value("${elasticsearch.index.alias:default}")
private String defaultAlias;